workers/unity/Packages/com.improbable.gdk.standardlibraryutils/CoordinatesExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Improbable;
+using UnityEngine;
+
+public static class CoordinatesExtensions
+{
+    /// <summary>
+    ///     Conversion method from Unity Vector3 to Spatial Coordinates.
```

Comment:
Nit: XMLDoc type names can be referred to using `<see cref=...`

Reply:
Unfortunately we don't put doc strings on code genned types. But can do it for `UnityEngine.Vector3`